hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/token/impl/AccountStore.java

```diff
@@ -116,4 +142,12 @@ private Long getAccountNum(final AccountID id) {
         }
         return id.getAccountNum();
     }
+
+    private KeyOrLookupFailureReason validateKey(final JKey key) {
+        if (key == null || key.isEmpty()) {
+            // need new response code ACCOUNT_IS_IMMUTABLE?
```

Comment:
Has this been addressed?

Reply:
Wanted to get feedback if we need new response code or re-use any existing response. If we need new response, I will update protobufs.

Reply:
Sure, so are we still waiting for this? 

In my n00b opinion I think `ACCOUNT_IS_IMMUTABLE` makes more sense, but could go either way on it

Reply:
Added `FUTURE:` comment on it.